protostar/testing/environments/fuzz_test_execution_environment.py

```diff
@@ -167,7 +199,7 @@ async def test(**inputs: Any):
         except InvalidArgument as ex:
             # This exception is sometimes raised by Hypothesis during runtime when user messes up
             # strategy arguments. For example, invalid range for `integers` strategy is caught here.
-            raise CheatcodeException("given", str(ex)) from ex
+            raise CheatcodeException("unknown", str(ex)) from ex
```

Comment:
[Functionality] Maybe `given or example` would be better to help the user with debugging.

Reply:
Hmm I think we should rather shift towards raising more general `FuzzingError` here.

Reply:
We can do that, although it doesn't look entirely right.

Reply:
`FuzzingError`? It actually can be right, depending on what perspective are you looking from.

Until now, we were pretending to say that strategies are statically fully determined during `given` cheatcode evaluation, which is **not** true. By raising `FuzzingError` we simply surface the fact that the strategies do some lazy initialization.

Reply:
Also, if we think about this more: strategies are silly only related to the `given` cheatcode 😅 

`example` operates on concrete values only.

Reply:
[change error message to more descriptive](https://github.com/software-mansion/protostar/pull/947/commits/a0e49741e12012e2a28febc07d675fadfad20b92)

Reply:
I find the use of `or` word confusing here and not professionally looking. Take into account that this string is presented as **cheatcode name** to the user! It'd be better to raise FuzzingError here, really IMHO

Reply:
Sounds good

Reply:
[raise fuzzing error instead of a cheatcode exception](https://github.com/software-mansion/protostar/pull/947/commits/aeabdb33a47b3da8c9377257b767f1347b5cba88)